app/assets/stylesheets/application.scss

```diff
@@ -113,11 +113,32 @@ li .join {
   background-color: #512DA8;
 }
 
+.back {
+  width: 140px;
+  margin: 0;
+}
+
 /* Datatables Styling */
 .dataTables_wrapper {
   max-height: 500px;
 }
 
+.dataTables_wrapper tr th:nth-child(2) {
+  max-width: 210px;
+}
+
+.dataTables_wrapper tr th:nth-child(3) {
```

Comment:
Merge rule `.dataTables_wrapper tr th:nth-child(3)` with rule on line 122<br>Selector should have depth of applicability no greater than 2, but was 3<br>Selector `dataTables_wrapper` should be written in lowercase with hyphens